### ISSUE TITLE
PSD topography opacity

### ISSUE THREAD

**Issue by lthimons** [ContactEngineering/TopoBank#626]
Just a small improvement that I think could be made. It seems that when I adjust the opacity value, regions where there is a lot of data overlapping is relatively unchanged. I assume this is because the opacity level is applied to each dataset, and having dozen of overlapping lines makes the area fairly opaque again. Is there some way to apply a uniform level of opacity regardless of how many lines are on top of one another?

**Comment by mcrot** [comment]
Yes, the opacity level is applied to each dataset, this is probably the reason. 

Interesting idea about the uniform level of opacity. I've looked around a bit in the [documentation of bokeh](https://docs.bokeh.org), the library I used for interactive plotting. So far I haven't found anything related, all alpha levels (opacity values) seem to be related to datasets. And we would need to built some kind of "group" for all measurements, because the 
lines for average should not be affected.

Maybe we could remove the lines and only display small dots, than they won't overlap.
Another question is, whether it would be easier to just have a radio button, which switches off all measurements.
